kernel-version.mk

```diff
@@ -33,8 +33,6 @@ ifeq ($(ZARCH), amd64)
     # either generic or rt
     KERNEL_FLAVOR=$(PLATFORM)
     KERNEL_VERSION=v6.1.38
-    KERNEL_COMMIT_generic=0c3194d84
```

Comment:
I think you can use the same logic that you've used for the commits and get rid of all these ifs, for instance:

```
KERNEL_arm64_nvidia_FLAVOR=nvidia
KERNEL_arm64_nvidia_VERSION=v6.1.38
KERNEL_riscv64__FLAVOR=generic
KERNEL_riscv64__VERSION=v6.1.38
```

Then the expansion of `KERNEL_$(ARCH)_$(PLATFORM)_FLAVOR` and `KERNEL_$(ARCH)_$(PLATFORM)_VERSION` will led automatically to the right values of FLAVOR and VERSION...

Reply:
@rene it is WAAAAY easier to generate all possible combination of ARCH-PLATFORM-FLAVOR based on branches in kernel repository. The same reason to have a separate file. I could do some template replacement magic but then we need a template file and update template file if necessary and probably the script to handle updated template. Now I just run the script and get kernel-commits.mk + commit message file.

Reply:
@rene Just to post here the result of our off-line discussion: I missunderstood your sugesstion. The sugesstion is to replace IF-ELSE block with the same variable expansion trick. However we'll have 16 variables covering current combinations. And even more in future. We agreed that if/else is more readable and clear